stripe/src/main/java/com/stripe/android/paymentsheet/BasePaymentMethodsListFragment.kt

```diff
@@ -32,6 +32,9 @@ internal abstract class BasePaymentMethodsListFragment : Fragment(
         )
     }
 
+    private var _viewBinding: FragmentPaymentsheetPaymentMethodsListBinding? = null
+    protected val viewBinding get() = requireNotNull(_viewBinding)
```

Comment:
why not just use lateinit var here? then it'd be non null adn we don't need custom getters

Reply:
Following the pattern here: https://developer.android.com/topic/libraries/view-binding#fragments